<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>553</width>
    <height>439</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QSlider" name="horizontalSliderEventoMin">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>100</y>
      <width>191</width>
      <height>31</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QLCDNumber" name="lcdNumberEventoMin">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>130</y>
      <width>71</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QTextBrowser" name="textBrowser">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>10</y>
      <width>256</width>
      <height>381</height>
     </rect>
    </property>
   </widget>
   <widget class="QLCDNumber" name="lcdNumberEventoMax">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>210</y>
      <width>71</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QSlider" name="horizontalSliderEventoMax">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>180</y>
      <width>191</width>
      <height>31</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonConnect">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>40</y>
      <width>121</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Connect</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonDesconnect">
    <property name="geometry">
     <rect>
      <x>149</x>
      <y>40</y>
      <width>121</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Desconnect</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonStart">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>360</y>
      <width>121</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Start</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonStop">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>360</y>
      <width>121</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Stop</string>
    </property>
   </widget>
   <widget class="QSlider" name="horizontalSliderTiming">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>280</y>
      <width>181</width>
      <height>20</height>
     </rect>
    </property>
    <property name="maximum">
     <number>10</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QLabel" name="labelMin">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>110</y>
      <width>21</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Min</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelMax">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>190</y>
      <width>21</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Max</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelTiming">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>270</y>
      <width>51</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Timing(s)</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditIP">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>261</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QLCDNumber" name="lcdNumberEventoTiming">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>290</y>
      <width>71</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>553</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>horizontalSliderEventoMin</sender>
   <signal>valueChanged(int)</signal>
   <receiver>lcdNumberEventoMin</receiver>
   <slot>display(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>200</x>
     <y>151</y>
    </hint>
    <hint type="destinationlabel">
     <x>270</x>
     <y>181</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>horizontalSliderEventoMax</sender>
   <signal>valueChanged(int)</signal>
   <receiver>lcdNumberEventoMax</receiver>
   <slot>display(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>153</x>
     <y>213</y>
    </hint>
    <hint type="destinationlabel">
     <x>220</x>
     <y>249</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>horizontalSliderTiming</sender>
   <signal>valueChanged(int)</signal>
   <receiver>lcdNumberEventoTiming</receiver>
   <slot>display(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>166</x>
     <y>305</y>
    </hint>
    <hint type="destinationlabel">
     <x>223</x>
     <y>334</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>copiaTexto()</slot>
 </slots>
</ui>
